Update configure-sharepoint-content-source.md (#633)
MicrosoftDocs/Viva: Viva/learning/configure-sharepoint-content-source.md

To configure SharePoint as a learning content sources in for Viva Learning, follow these steps:

> 1. Open Viva Learning in Microsoft Teams, go to **Admin** > **Manage Providers**.

Reviewer: ```suggestion
1. Open Viva Learning in Microsoft Teams and go to **Admin** > **Manage Providers**.
```

<img width="1510" alt="image" src="https://github.com/MicrosoftDocs/Viva/assets/86094924/cab8c569-39b2-4880-9096-1cab51f2b9b3">

> 3. Click Add Provider, select Share Point and click Next.

Reviewer: ```suggestion
3. Select **Add Provider** > **Share Point** > **Next**.
```

<img width="605" alt="image" src="https://github.com/MicrosoftDocs/Viva/assets/86094924/dc1b7523-c7e2-4f88-9473-28f6dcdf39f3">

> 4. Set a Display name, provide the site URL to the SharePoint site where you want Viva Learning to create a centralized repository, click Save. If your SharePoint site is new, you'll need to wait 1 hour after site creation to add it here. You must also be the owner of the SharePoint site.

Reviewer: ```suggestion
4. Set a display name, provide the site URL to the SharePoint site where you want Viva Learning to create a centralized repository, and select **Save**. If your SharePoint site is new, you'll need to wait one hour after site creation to add it here. You must also be the owner of the SharePoint site.
```
